static/js/app.js

```diff
@@ -136,6 +158,11 @@ function addEventListeners() {
 			sendGaPing("Create New Relay Alias", "Click", createNewRelayBtn.dataset.analyticsLabel);
 		})
 	});
+
+	const continueWithoutAddonBtn = document.querySelector(".continue-without-addon");
+	if (continueWithoutAddonBtn) {
+		continueWithoutAddonBtn.addEventListener("click", hideInstallCallout());
```

Comment:
`hideInstallCallout()` calls the function immediately. Need to drop the `()`.

```suggestion
		continueWithoutAddonBtn.addEventListener("click", hideInstallCallout);
```